frontend/src/metadata/MetadataPanel.svelte

```diff
@@ -327,6 +329,22 @@
 			</div>
 		</div>
 		<div class="inline">
+			<div
+				use:tooltip={{
+					content: "Discover underperforming slices.",
+					position: "bottom",
+					theme: "zeno-tooltip",
+				}}>
+				<IconButton
+					on:click={() => {
+						showNewFolder.set(false);
+						showSliceFinder.update((c) => !c);
+					}}>
+					<Icon component={Svg} viewBox="0 0 24 24">
+						<path fill="black" d={mdiAssistant} />
```

Comment:
Can we use creationOutline here:

import { mdiCreationOutline } from '@mdi/js';